### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by ghost**
Task:
https://github.com/akvatol/moffopt/projects/1#card-83800393

### CODE REVIEW

File: .github/workflows/github-actions.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: CI
+on: [push, 	pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest  # TODO specify disto and version
```
Comment (akvatol) [inline .github/workflows/github-actions.yml]: Close all TODO`s before you try to merge something into the main branch

File: .github/workflows/github-actions.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    strategy:
+      matrix:
+        python-version: ["3.10"]  # TODO specify python version
```
Comment (ghost) [inline .github/workflows/github-actions.yml]: @akvatol, which python version We will use?

Comment (akvatol) [reply .github/workflows/github-actions.yml]: 3.10

File: .github/workflows/github-actions.yml (hunk 1 above)
Comment (ghost) [reply .github/workflows/github-actions.yml]: done

File: .github/workflows/github-actions.yml (hunk 2 above)
Comment (ghost) [reply .github/workflows/github-actions.yml]: done, also added 3.7

**Review by akvatol — DISMISSED** [review]

**Review by akvatol — APPROVED** [review]

**Pull request merged**